foodsaving/groups/emails.py

```diff
@@ -69,6 +69,11 @@ def prepare_group_summary_data(group, from_date, to_date):
         'settings_url': settings_url,
     }
 
+    data['has_activity'] = any(data[field] > 0 for field in ['pickups_done_count', 'pickups_missed_count']) or \
+        any(len(data[field]) > 0 for field in ['feedbacks', 'messages', 'new_users'])
```

Comment:
It would seem slightly nicer to filter for no activity higher up with the orm objects rather than the output data structures so there is more certainty that the types it is checking is correct.